Feature/card

kalleericson opened the pull request:
Card feature, as run by the Statistics project.
Please check structure is in line and OK.

knegusen commented on src/components/ui/card/Card.tsx:
```diff
@@ -0,0 +1,173 @@
+import * as React from 'react';
```
Could this one be split up in more components? It's quite big right now.

kalleericson commented:
<img width="822" alt="screen shot 2018-08-22 at 10 44 29" src="https://user-images.githubusercontent.com/7943407/44455557-af87f800-a5fe-11e8-814a-766e8ba8fd00.png">

- Split "Card" component into subcomponents "Header, HeaderButton, HeaderNotice, Content". 
- Updated test suite to match new structure.
- Added screenshot

kalleericson commented:
Yeah, I'm aware and have changed structure of all tests written _after_ our discussion. However, I didn't change the tests already written. From now on, all is in order!

Great!

knegusen commented:
Merge this?
